src/ban.cpp

```diff
@@ -44,15 +44,15 @@ bool Ban::acceptConnection(const Connection::Address& clientIP)
 	return true;
 }
 
-bool IOBan::isAccountBanned(uint32_t accountId, BanInfo& banInfo)
+const std::optional<BanInfo> IOBan::isAccountBanned(uint32_t accountId)
```

Comment:
The methods should be called: `getMethodName` instead of `isMethodName`.

Reply:
_has_ may be better than _get_

Reply:
both ugly 😄

Reply:
Why not call them `getAccountBanInfo` and `getIPBanInfo`?